### PR TITLE
docs: explain how ngx-template-streams works with typescript strict mode

### PR DESCRIPTION
**Pull request by GregOnNet**
Hi,

according to #6 I added a small explanation to the FAQ section.
Please let me know if I can provide any further information.

Kinds
Greg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -399,6 +399,26 @@ to follow the [manual installation guide](#manual-installation) instead of using
 
 No, that's not necessary. The library will automatically complete and clean up every event stream for you when the component is destroyed.
 
+### What if I use typescript's strict mode?
```
Comment (d3lm) [inline README.md]: Maybe capitalize `Typescript` here?

File: README.md [hunk 2]
```diff
@@ -399,6 +399,26 @@ to follow the [manual installation guide](#manual-installation) instead of using
...
+
+If you are working with TypeScript's strict mode or have set strictPropertyInitialization to true in your tsconfig.json, you will experience an error using @ObservableEvent().
```
Comment (d3lm) [inline README.md]: Use inline code blocks for `strictPropertyInitialization`, `tsconfig.json` and `@ObservableEvent()`

**Review by d3lm — CHANGES_REQUESTED** [review]
Great addition, thanks! Left a few minor comments.

**Comment by GregOnNet** [comment]
Thanks for the comments.
I fixed the description accordingly.

**Review by d3lm — APPROVED** [review]
Looks good! Thanks @GregOnNet 👍

**Pull request merged**